ops/ansible/playbooks-ccs/files/prod-sbx-bluebutton-appserver-public-cert.pem

```diff
@@ -0,0 +1,34 @@
+-----BEGIN CERTIFICATE-----
+MIIF7DCCA9SgAwIBAgIEfh/MBjANBgkqhkiG9w0BAQsFADAfMR0wGwYDVQQDExRw
+cm9kLXNieC5iZmQuY21zLmdvdjAeFw0xOTA5MTgwMjAyMzdaFw0yMTA5MTcwMjAy
+MzdaMB8xHTAbBgNVBAMTFHByb2Qtc2J4LmJmZC5jbXMuZ292MIICIjANBgkqhkiG
+9w0BAQEFAAOCAg8AMIICCgKCAgEAop2rYkkbHeOISOq3X7frNni/u11vYhHVxdbU
+XE2BzJi7Di0JkGohRmwibu8FIn1oPclk88Qh58Xf1Rtc+cLjvDOjlMWLx4z+vhG2
+PgmhMuObhFjjz/V2oUCHsrhiSF86foTPnZeWra5bJWD30uCnnI+oHTeQyp8UOSwO
+T4r93qwnWn3NZ4cDg7ptYzVDy9hIs/n7fM+oMvL4oHQxXqBw8F85Uzw8Sy12Avpa
+NUj62u3msYQ2MtyDf6O3j+Xv9M3jTgYQOHBqd8Z2ba7j3QKqr8JmYk8NpkJYfBWr
+7KaVZXhe44XssDcb3aIqvzIIberc9AZkr339EGj7IG6XaqKFf893XL4lZx+vYV66
+HnvRcdp5QKiR7JJmyrGE7MG4wlt+ddUeizU/gOdZ+5k0ud8xgC0T1c7Bn3YTLygl
+g7bkctfh5UtVyX87bYLiL6+1blSfJt5GZ1NfrkmN3Aq+NVZpU8kna1YV8HoFCvra
+IEfI9eWepyN3o04tMyJylIkd6n4VgjYqmGme45XYijuvyl4oc5oYpeqsRJWIvAtP
+y4/EJzppRxVdYSeTECfWtMGe5WLlWfVm7jpl9wzf5CT4YyZOAwVDmql1Y3h0Q4J5
+AEzac80UvKHJXlGZo/z42dCW4221ItOp9DzUhiy3HEmT+mNTJoUL22M/3BsRLxa0
+JHvVstkCAwEAAaOCAS4wggEqMIIBBwYDVR0RBIH/MIH8ghVwcm9kLXNieC5iZmRj
+bG91ZC5uZXSCGW1jdC5wcm9kLXNieC5iZmRjbG91ZC5uZXSCGWRwYy5wcm9kLXNi
+eC5iZmRjbG91ZC5uZXSCGmJjZGEucHJvZC1zYnguYmZkY2xvdWQubmV0ghhiYi5w
+cm9kLXNieC5iZmRjbG91ZC5uZXSCOWludGVybmFsLWRwY3dlbGIwMS0yMDc0MDcw
+ODY4LnVzLWVhc3QtMS5lbGIuYW1hem9uYXdzLmNvbYIUcHJvZC1zYnguYmZkLmNt
+cy5nb3aCJmZoaXIuYmFja2VuZC5ibHVlYnV0dG9uLmhoc2RldmNsb3VkLnVzMB0G
+A1UdDgQWBBRlxirL5JJ52lEX5vS6QujWQ8hYVTANBgkqhkiG9w0BAQsFAAOCAgEA
+oHFKlX9YwUO9BL4BEPi5uHXDeU+mS2jFTYULQ45LIi/hE87y5xNbwmV3RH4QP3/g
+hC0Sk/03sQCrrL5BE4s/WaZj0MLvQPYcrN3/58DKsX/ogg+9wfYcU9IRCU55LnVM
+5N5ey7pTH+l7Id35bxzKRAY8nJjIRO9A7PLqV9n5yDCpJZSHeiXstkfR53jPIWmL
+U1snNrBOcG4yUycWVAfr+qaNES9dHww7l0NtFTzTJKHDlqgahpvEz30RZf07/bVv
+9xSLhFWFo3efKA0Y6YL/cVmX3Onv/fhUalkHj2mTAXiYGKQNDNPFUmjM1lL387AU
+3Sdb5woAZnBtpoO8+iqSHTkaaYbwukIkOVc8tcg4cE2maOnOz1JRRFHwnNzSnCmw
+TeBwGVvBQC60CrN+/PRWgdMuv+i+KB//ZrtseZOvQ4fUUZ1qyJcrrvaX04Vt6SAj
+bnkdEDY4NtLzGi7/HAvUMEnJPkTg3CopcYghNjyPyv/fIDnPZTjpfqWJHWtD7/5K
+gy8S2HyH2/OQKaqltGI5+rXMyTzmJ8j9W32tj2Kw2LwpcOjfulHQgOxWeP+/Vvki
+MULDJ/6IbjortwrrTlnF/PFShJKBOw7900zziZ34983AJHjJvpN61G29uhrkstLn
+CA++ERlZF9kzSMepxqJKTJMVprEzpbwh9IpH16fhFGw=
+-----END CERTIFICATE-----
```

Comment:
Reviewing the cert:

```
$ openssl x509 -text -in ops/ansible/playbooks-ccs/files/prod-sbx-bluebutton-appserver-public-cert.pem | less
Certificate:
    Data:
        Version: 3 (0x2)
        Serial Number: 2116013062 (0x7e1fcc06)
    Signature Algorithm: sha256WithRSAEncryption
        Issuer: CN=prod-sbx.bfd.cms.gov
        Validity
            Not Before: Sep 18 02:02:37 2019 GMT
            Not After : Sep 17 02:02:37 2021 GMT
        Subject: CN=prod-sbx.bfd.cms.gov
        Subject Public Key Info:
            Public Key Algorithm: rsaEncryption
                Public-Key: (4096 bit)
                Modulus:
                    00:a2:9d:ab:62:49:1b:1d:e3:88:48:ea:b7:5f:b7:
...
                Exponent: 65537 (0x10001)
        X509v3 extensions:
            X509v3 Subject Alternative Name:
                DNS:prod-sbx.bfdcloud.net, DNS:mct.prod-sbx.bfdcloud.net, DNS:dpc.prod-sbx.bfdcloud.net, DNS:bcda.prod-sbx.bfdcloud.net, DNS:bb.prod-sbx.bfdcloud.net, DNS:internal-dpcwelb01-2074070868.us-east-1.elb.amazonaws.com, DNS:prod-sbx.bfd.cms.gov, DNS:fhir.backend.bluebutton.hhsdevcloud.us
            X509v3 Subject Key Identifier:
                65:C6:2A:CB:E4:92:79:DA:51:17:E6:F4:BA:42:E8:D6:43:C8:58:55
    Signature Algorithm: sha256WithRSAEncryption
         a0:71:4a:95:7f:58:c1:43:bd:04:be:01:10:f8:b9:b8:75:c3:
...
```

Looks good to me.

For comparison, here's the old DPR cert:

```
$ openssl x509 -text -in ops/ansible/playbooks-healthapt/files/server_data_server_env_dpr_certificate.pem | less
Certificate:
    Data:
        Version: 3 (0x2)
        Serial Number: 632269137 (0x25afa951)
    Signature Algorithm: sha256WithRSAEncryption
        Issuer: CN=internal-dpcwelb01-2074070868.us-east-1.elb.amazonaws.com
        Validity
            Not Before: Mar  1 20:39:14 2019 GMT
            Not After : Feb 28 20:39:14 2022 GMT
        Subject: CN=internal-dpcwelb01-2074070868.us-east-1.elb.amazonaws.com
        Subject Public Key Info:
            Public Key Algorithm: rsaEncryption
                Public-Key: (4096 bit)
                Modulus:
                    00:99:3c:ab:66:57:d1:2e:99:e8:17:3b:19:a2:99:
...
                Exponent: 65537 (0x10001)
        X509v3 extensions:
            X509v3 Subject Key Identifier:
                37:F6:E9:F9:79:B5:12:44:BE:5B:BD:6E:54:F1:C4:6C:1C:17:F7:FF
    Signature Algorithm: sha256WithRSAEncryption
         67:5b:30:d8:f3:23:fd:c6:be:d6:ae:9b:a2:16:40:05:47:9e:
...
```